scipy/stats/_distribution_infrastructure.py

```diff
@@ -0,0 +1,5082 @@
+    # each of these private methods calls a private "dispatch" method that
+    # determines which "method" (strategy for calculating the desired quantity)
+    # to use. Also, each dispatch method can designate the responsibility
+    # computing the moment to one of several "implementation" methods.
+    # Unlike the dispatch methods above, however, the `@_dispatch` decorator
+    # is not used, and both logic and method calls are included in the function
+    # itself.
+    # Instead of determining which method will be used based solely on the
+    # implementation methods available and calling only the corresponding
+    # implementation method, *all* the implementation methods are called
+    # in sequence until one returns the desired information. When an
+    # implementation methods cannot provide the requested information, it
+    # returns the object None (which is distinct from arrays with NaNs or infs,
+    # which are valid values of moments).
+    # The reason for this approach is that although formulae for the first
+    # few moments of a distribution may be found, general formulae that work
+    # for all orders are not always easy to find. This approach allows the
+    # developer to write "formula" implementation functions that return the
+    # desired moment when it is available and None otherwise.
+    #
+    # Note that the first implementation method called is a cache. This is
+    # important because lower-order moments are often needed to compute
+    # higher moments from formulae, so we eliminate redundant calculations
+    # when moments of several orders are needed.
+
+    @cached_property
+    def _moment_methods(self):
+        return {'cache', 'formula', 'transform',
+                'normalize', 'general', 'quadrature'}
+
+    @property
+    def _zero(self):
+        return self._constants()[0]
+
+    @property
+    def _one(self):
+        return self._constants()[1]
+
+    def _constants(self):
+        if self._constant_cache is not None:
+            return self._constant_cache
+
+        constants = self._preserve_type([0, 1])
+
+        if self.cache_policy != _NO_CACHE:
+            self._constant_cache = constants
+
+        return constants
+
+    @_set_invalid_nan_property
+    def moment(self, order=1, kind='raw', *, method=None):
+        r"""Raw, central, or standard moment of positive integer order
+
+        In terms of probability density function :math:`f(x)` and its support
+        :math:`\chi`, the "raw" moment (about the origin) of order :math:`n` of
+        a random variable :math:`X` is:
+
+        .. math::
+
+            \mu'_n(X) = \int_{\chi} x^n f(x) dx
+
+        The "central" moment is the raw moment taken about the mean,
+        :math:`\mu = \mu'_1`:
+
+        .. math::
+
+            \mu_n(X) = \int_{\chi} (x - \mu) ^n f(x) dx
+
+        The "standardized" moment is the central moment normalized by a power of
+        the standard deviation, :math:`\sigma = \sqrt{\mu'_2}`:
```

Comment:
`\sigma = \sqrt{\mu_2}`, not `\sigma = \sqrt{\mu'_2}`

Reply:
Thanks! Fixed.

Reply:
can be resolved